Library/Formula/bazel.rb

```diff
@@ -0,0 +1,25 @@
+class Bazel < Formula
+  homepage "http://bazel.io/"
+  url "https://github.com/google/bazel.git", :tag => "0.0.2"
+  sha256 "5865356e74622c248248174acd09f38c07f3983867d7e687fa5932f2216e84c0"
+
+  head "https://github.com/google/bazel.git"
+
+  depends_on "protobuf" => :build
+  depends_on "libarchive"
+
+  def install
+    if build.stable?
+      inreplace "compile.sh", "$(brew --prefix 2>/dev/null)/Cellar/libarchive/*/include/archive.h", "#{Formula["libarchive"].opt_include}/archive.h"
+    else
+      inreplace "compile.sh", "$(brew --prefix libarchive 2>/dev/null)/include/archive.h", "#{Formula["libarchive"].opt_include}/archive.h"
+    end
+
+    system "./compile.sh"
+    bin.install "./output/bazel"
+  end
+
+  test do
+    #TODO: tests; see https://github.com/Homebrew/homebrew/pull/38061#discussion-diff-27137134
```

Comment:
A good call would be to call the `bazel` too and write a simple thing for it to compile.